.env.example

```diff
@@ -41,6 +41,7 @@ MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
 
 VATUSA_FACILITY=ZTL
 VATUSA_API_KEY=
+VATUSA_API_BASE=https://api.vatusa.net
```

Comment:
I understand what you are trying to do here, but this change doesn't have anything to do with this particular PR. Please open up a new issue that addresses this change and move the code changes for making the VATUSA API base URL into an environment variable to the new issue. We want to keep these separate as much as possible - that way if there is an issue when we deploy we can roll back code changes appropriately.

Reply:
Removed